api/views/assets.py

```diff
@@ -123,19 +123,27 @@ def get_queryset(self):
 class AssetViewSet(ModelViewSet):
     serializer_class = AssetSerializer
     permission_classes = [IsAuthenticated]
+    queryset = models.Asset.objects
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ["get"]
 
     def get_queryset(self):
         user = self.request.user
         query_filter = {}
-
         if not self.request.user.is_securityuser:
```

Comment:
@kmazi  A question, any particular reason to have security users exempted from this request? All requests from a security user keep returning the entire asset dataset rather than filtering it to list assets for a particular user as specified in the URL.

Reply:
@egichuri